.lintr

```diff
@@ -1,5 +1,6 @@
 linters: linters_with_defaults(
   line_length_linter = line_length_linter(120),
   cyclocomp_linter = NULL,
-  object_usage_linter = NULL
+  object_usage_linter = NULL,
+  indentation_linter = NULL
```

Comment:
@kartikeyakirar in some packages you edit that, and in some you dont?

Reply:
I've made edits where the checks were failing, but I believe this issue should be addressed more thoroughly. Indentation problems are affecting most packages.

Reply:
@kartikeyakirar I rather have you update the code, so that we follow new indentation rules, instead of omitting the check.
Please remove this indentation NULL in .lintr and please edit the code where we need to remove or append lines. Sorry.
There should be a lintr function that should change indentation for you, so this should be one run of one function. Thanks

Reply:
ok we had a team call. lets leave it as it is with indentation_linter = NULL